changelog.json

```diff
@@ -1,5 +1,22 @@
 {
   "releases": {
+    "2.5.3": [
+      "[Added] Stash changes without switching branches - #10032",
+      "[Added] Discard selection of lines from a file - #9728",
+      "[Fixed] Display more readable messages for errors when cloning a repository - #9944",
+      "[Fixed] Support launching Desktop from WSL on Windows - #4998. Thanks @shawnfunke!",
+      "[Fixed] Select correct branch to compare when two branches have similar names - #9977",
+      "[Fixed] Make Ctrl/Cmd+W shortcut work in all dialogs - #9826. Thanks @ruggi99!",
+      "[Fixed] Fix links when text wrapping is required in commit summaries - #9185",
+      "[Fixed] Prevent crash when an opened repository doesn't have an owner on GitHub - #9895",
+      "[Fixed] Sign in correctly when pressing \"Enter\" on the username/password authentication form - #9899",
+      "[Fixed] Maintain selected files when navigating back to the Changes tab - #7710. Thanks @TerryChan!",
+      "[Fixed] Maintain selected files when deleting a stash - #9921",
+      "[Improved] Remove redundant progress text from Git error messages - #9945",
+      "[Improved] Remove flicker when switching between files in the changes tab - #9929",
+      "[Improved] Remove space between the number and the percent sign when showing the progress of a clone operation - #9893",
```

Comment:
From 7e2e48b36d1d6dc5a83390eabef2270ba9f94a8c Mon Sep 17 00:00:00 2001 From: Markus Olsson <j.markus.olsson@gmail.com> Date: Tue, 23 Jun 2020 12:07:22 +0200 Subject: [PATCH 1/2] bumpety bump --- app/package.json | 2 +- 1 file changed, 1 insertion(+), 1 deletion(-) diff --git a/app/package.json b/app/package.json index 66ca62c89f..85fb110e59 100644 --- a/app/package.json +++ b/app/package.json @@ -3,7 +3,7 @@ "productName": "GitHub Desktop", "bundleID": "com.github.GitHubClient", "companyName": "GitHub, Inc.", - "version": "2.5.3-beta2", + "version": "2.5.3", "main": "./main.js", "repository": { "type": "git", From f26bc4faa3aa22d7d75c3dbe17171ffe69e04b86 Mon Sep 17 00:00:00 2001 From: Markus Olsson <j.markus.olsson@gmail.com> Date: Tue, 23 Jun 2020 12:07:37 +0200 Subject: [PATCH 2/2] Initial changelerg --- changelog.json | 17 +++++++++++++++++ 1 file changed, 17 insertions(+) diff --git a/changelog.json b/changelog.json index 63d09d4105..64643be982 100644 --- a/changelog.json +++ b/changelog.json @@ -1,5 +1,22 @@ { "releases": { + "2.5.3": [ + "[Added] Stash changes without switching branches - #10032", + "[Added] Discard selection of lines from a file - #9728", + "[Fixed] Display more readable messages for errors when cloning a repository - #9944", + "[Fixed] Support launching Desktop from WSL on Windows - #4998. Thanks @shawnfunke!", + "[Fixed] Select correct branch to compare when two branches have similar names - #9977", + "[Fixed] Make Ctrl/Cmd+W shortcut work in all dialogs - #9826. Thanks @ruggi99!", + "[Fixed] Fix links when text wrapping is required in commit summaries - #9185", + "[Fixed] Prevent crash when an opened repository doesn't have an owner on GitHub - #9895", + "[Fixed] Sign in correctly when pressing \"Enter\" on the username/password authentication form - #9899", + "[Fixed] Maintain selected files when navigating back to the Changes tab - #7710. Thanks @TerryChan!", + "[Fixed] Maintain selected files when deleting a stash - #9921", + "[Improved] Remove redundant progress text from Git error messages - #9945", + "[Improved] Remove flicker when switching between files in the changes tab - #9929", + "[Improved] Remove space between the number and the percent sign when showing the progress of a clone operation - #9893", + "[Improved] Automatically disable username and password authentication when the API no longer supports it - #9231" + ], "2.5.3-beta2": [ "[Added] Stash changes without switching branches - #10032", "[Fixed] Support launching Desktop from WSL on Windows - #4998. Thanks @shawnfunke!",